SerialPrograms/Source/CommonFramework/OCR/TesseractPA.h

```diff
@@ -92,7 +142,6 @@ class TesseractString{
         : m_str(str)
     {}
 
-private:
```

Comment:
Why remove this?

Reply:
Oh, there is another `private:` above this line, so this `private:` does nothing. I noticed in multiple places of the project there are redundant `private:` or `public:` keywords in classes. Is this some kind of coding feature?

Reply:
Change the code back in the new commit. Resolve the question in Discord discussion.